operator/tests/securedcluster/basic-sc/40-assert.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: central-prometheus-k8s
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: central-prometheus-k8s
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: rhacs-central-auth-reader
+  namespace: kube-system
+---
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: central-monitoring-tls
```

Comment:
Why the `-tls` suffix?

Reply:
https://github.com/stackrox/stackrox/blob/7878c65e42f41ec2de4dee5747c671fd35f5f429/image/templates/helm/stackrox-central/templates/01-central-10-networkpolicy.yaml#L46-L49

We added new TLS endpoint just for OpenShift Monitoring to not expose unsecured metrics by default.

Current dependencies on/for this PR:
* master
  * **PR #6665** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/6665" target="_blank"><img src="https://static.graphite.dev/graphite-32x32.png" alt="Graphite" width="10px" height="10px"/></a> 
    * **PR #6666** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/6666" target="_blank"><img src="https://static.graphite.dev/graphite-32x32.png" alt="Graphite" width="10px" height="10px"/></a>  👈
      * **PR #6667** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/6667" target="_blank"><img src="https://static.graphite.dev/graphite-32x32.png" alt="Graphite" width="10px" height="10px"/></a> 
        * **PR #6668** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/6668" target="_blank"><img src="https://static.graphite.dev/graphite-32x32.png" alt="Graphite" width="10px" height="10px"/></a> 
          * **PR #6796** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/6796" target="_blank"><img src="https://static.graphite.dev/graphite-32x32.png" alt="Graphite" width="10px" height="10px"/></a> 
          * **PR #6669** <a href="https://app.graphite.dev/github/pr/stackrox/stackrox/6669" target="_blank"><img src="https://static.graphite.dev/graphite-32x32.png" alt="Graphite" width="10px" height="10px"/></a>

Reply:
Just a thought about non-acscs installations. 
Is it a requirement that all openshift monitoring (e.g. on prem) use TLS? If not, would be safe to assume we can remove the `-tls` suffix

Reply:
> Is it a requirement that all openshift monitoring (e.g. on prem) use TLS? 
It's not required but AFAIK it's recommended so we created dedicated endpoint and exposed it if monitoring is enabled for all OpenShift 4 installations (3 does not support certificates).

Reply:
We already have a non-TLS monitoring endpoint that is opt-in and may be used for custom Prometheus setups. However the OpenShift Prometheus will use the TLS one.

Reply:
Does that mean that we can potentially have 2 separate ServiceMonitors for the same central service ?

Reply:
No, the non-TLS monitoring flag does not create a service monitor, just exposes the endpoint. There could be multiple service monitors if multiple Central charts are installed, however the service monitor have namespace selectors.

Reply:
Thanks for the reply. And what is the difference between "normal" prometheus monitoring and "openshift" monitoring ?

Reply:
openshift monitoring is part of openshift installation. after having metrics there we can pass them further to [telemeter](https://docs.openshift.com/container-platform/4.8/support/remote_health_monitoring/about-remote-health-monitoring.html) and get telemetry data in "official" RedHat way. 

"normal" prometheus is prometheus installed and configured by user. It might happen that it's openshift prometheus but it also could be external to a cluster.

Reply:
Ah just realized this is the secured cluster PR not the Central PR, so pretty much `s/central/securedcluster`.

Reply:
Should the network policies / rbac etc. be also included for the SecuredCluster? Seems like the PR title doesn't represent the changes

Reply:
They should, I think the asserts are wrong. We have a set of Central resources, and a set of secured cluster resources.